### PR TITLE
Cockpit 220

### PR DESCRIPTION
**Pull request by marusak**
Preview: https://marusak.dev/cockpit-project.github.io/blog/cockpit-220.html

### CODE REVIEW

**Comment by marusak** [comment]
First of all, commit message is wrong :)

File: _posts/2020-05-27-cockpit-220.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 220.
+
+### New navigation with integrated hosts switching
+
+The navigation has been redesigned and brings four major improvements:
+- **One level navigation**: The current two level navigation has been squashed for better use of space and better discoverability
+- **Integrated hosts switching**: Switching between hosts as well as editing them can be done now directly from the navigation without using the 'Dashboard' component
+- **Better discoverability of applications**: Applications are shown as the first group in the menu and are also searchable
+- **Super indicator stuff**: Yeah, lets mention #14055 here
```
Comment (marusak) [inline _posts/2020-05-27-cockpit-220.md:20]: @mvollmer

File: _posts/2020-05-27-cockpit-220.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+summary: Cockpit with new navigation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 220.
+
+### New navigation with integrated hosts switching
+
+The navigation has been redesigned and brings four major improvements:
+- **One level navigation**: The current two level navigation has been squashed for better use of space and better discoverability
```
Comment (martinpitt) [inline _posts/2020-05-27-cockpit-220.md:17]: I think it's "two-level"

File: _posts/2020-05-27-cockpit-220.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- **Integrated hosts switching**: Switching between hosts as well as editing them can be done now directly from the navigation without using the 'Dashboard' component
+- **Better discoverability of applications**: Applications are shown as the first group in the menu and are also searchable
+- **Super indicator stuff**: Yeah, lets mention #14055 here
+
+![New one level navigation](/images/one-level-nav.png)
```
Comment (martinpitt) [inline _posts/2020-05-27-cockpit-220.md:22]: Screenshot is too big IMHO, and scaled down. How about cropping out just the nav menu and a little bit of the overview page, and then putting the host switcher one side by side?

File: _posts/2020-05-27-cockpit-220.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+![Integrated host switcher](/images/host-switcher.png)
+
+### Logs: Inline help for filtering
+
+In the [previous release](https://cockpit-project.org/blog/cockpit-219.html) new possibilities how to search for logs were introduced.
```
Comment (martinpitt) [inline _posts/2020-05-27-cockpit-220.md:27]: This is a bit hard to read -- how about "The previous release introduced ..."?

File: _posts/2020-05-27-cockpit-220.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+Now is also provided inline overview of accepted options and `journalctl` command which shows the same log entries in terminal.
```
Comment (martinpitt) [inline _posts/2020-05-27-cockpit-220.md:28]: This release adds a help button that shows an overview of accepted options, and the `journalctl` command corresponding to the current filter.

File: _posts/2020-05-27-cockpit-220.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+![Logs inline help](/images/logs-inline-help.png)
+
+### Storage: Improve side panel on details page
+
+Side panel on storage details page has been unified and uses the same layout as on the storage overview page.
+
+![Storage side panel](/images/storage-details-side-panel.png)
```
Comment (martinpitt) [inline _posts/2020-05-27-cockpit-220.md:36]: Let's make this more narrow to avoid scaling down.

File: _posts/2020-05-27-cockpit-220.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2020-05-27-cockpit-220.md:20]: "Access level for all hosts: You can change between Administrative and Limited access on every host, right from the navigation."

File: _posts/2020-05-27-cockpit-220.md (hunk 3 above)
Comment (marusak) [reply _posts/2020-05-27-cockpit-220.md:22]: The top right corner changed as well, I want that in screenshot as well. But I can make it bigger and the menu more compact to have better screenshots

Comment (martinpitt) [reply _posts/2020-05-27-cockpit-220.md:22]: Ah, fair enough -- I usually make my browser around 1000px wide for screenshots (i. e. as narrow as possible to still see everything decently, but avoiding useless whitespace), not the full 1920.

**Comment by marusak** [comment]
Updated

File: _posts/2020-05-27-cockpit-220.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+tags: cockpit linux
+slug: cockpit-220
+category: release
+summary: Cockpit with new navigation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 220.
+
+### New navigation with integrated hosts switching
```
Comment (martinpitt) [inline _posts/2020-05-27-cockpit-220.md:14]: I think "host switching" (singular) is easier to read.

File: _posts/2020-05-27-cockpit-220.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+summary: Cockpit with new navigation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 220.
+
+### New navigation with integrated switching of hosts
+
+The navigation has been redesigned and brings four major improvements:
+- **One level navigation**: The current two-level navigation has been squashed for better use of space and better discoverability
```
Comment (martinpitt) [inline _posts/2020-05-27-cockpit-220.md:17]: Let's be consistent then and do "One-level navigation" as well.

File: _posts/2020-05-27-cockpit-220.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+- **Integrated hosts switching**: Switching between hosts as well as editing them can be done now directly from the navigation without using the 'Dashboard' component
+- **Better discoverability of applications**: Applications are shown as the first group in the menu and are also searchable
+- **Access level for all hosts**: You can change between Administrative and Limited access on every host, right from the navigation
+
+![New one level navigation](/images/one-level-nav.png) | ![Integrated host switcher](/images/host-switcher.png)
+
+### Logs: Inline help for filtering
+
+The [previous release](https://cockpit-project.org/blog/cockpit-219.html) introduced new possibilities how to search for logs.
```
Comment (allisonkarlitskaya) [inline _posts/2020-05-27-cockpit-220.md:26]: "introduced new possibilities how to" doesn't scan

"for how to" is more correct, but sounds even worse.

how about "introduced new advanced search features for logs"?

File: _posts/2020-05-27-cockpit-220.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+This release adds a help button that shows an overview of accepted options, and the `journalctl` command corresponding to the current filter.
+
+![Logs inline help](/images/logs-inline-help.png)
+
+### Storage: Improve side panel on details page
+
+Side panel on storage details page has been unified and uses the same layout as on the storage overview page.
```
Comment (allisonkarlitskaya) [inline _posts/2020-05-27-cockpit-220.md:33]: +The side panel on +the storage details page.

**Pull request merged**